src/MockBot.Api/Models/Chat.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MockBot.Api.Models;
+
+public class Chat
+{
+    public Guid Id { get; set; } = Guid.NewGuid();
+
+    [Required] public List<Message> Messages { get; set; } = new List<Message>();
+}
+
+public class Message
+{
+    private Guid Id { get; set; } = Guid.NewGuid();
```

Comment:
I am assuming we do not need the 'get' property here since all the members are private ?

Reply:
I did end up changing all of these to public.